CMakeLists.txt

```diff
@@ -15,10 +15,15 @@ endif()
 set(CMAKE_CXX_STANDARD 14)
 #Ensures MSVC treats strings as UTF-8 at both compile/runtime time
 if (MSVC)
+	add_compile_options("/D_WIN32_WINNT=0x0600" "/wd4141" "/wd4018")
```

Comment:
Should `_WIN32_WINNT` defined through `if(WIN32)` while specific flags with the same `if(MSVC)`?

Reply:
/D vs -D problems :p

Reply:
you can use `add_definitions` where CMake handles it but well i suppose these IFs are nearby so it is not a problem